docs/userguide.md

```diff
@@ -247,6 +247,72 @@ done
 If you were able to verify the resources removed and added back then you have
 successfully verified a working federation-v2 deployment.
 
+In addition to specifying an explicit list of clusters that a resource should be
+propagated to via the `spec.clusterNames` field of a placement resource, it is possible
+to use the `spec.clusterSelector` field to provide a label selector that determines
+that list of clusters dynamically.
+
+If the goal is to select a subset of member clusters, make sure that the `FederatedCluster`
+resources that are intended to be selected have the appropriate labels applied.
+
+The following command is an example to label a `FederatedCluster`:
+
+```bash
+kubectl label federatedclusters -n federation-system cluster1 foo=bar
+```
+
+Please refer to [Kubernetes label command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#label)
+to get more detail for how `kubectl label` works.
+
+The following sections detail how `spec.clusterNames` and `spec.clusterSelector` are
+used in determining the clusters that a federated resource should be propagated to.
+
+#### Neither `spec.clusterNames` nor `spec.clusterSelector` is provided
+
+```yaml
+spec: {}
+```
+
+In this case, you can either set `sepc: {}` as above or remove `spec` section from your
+federate placement policy. The resource will not be propagated to member clusters.
+
+#### Both `spec.clusterNames` and `spec.clusterSelector` are provided
+
+```yaml
+spec:
+  clusterNames:
+    - cluster2
+    - cluster1
+  clusterSelector:
+    matchLabels:
+      foo: bar
+```
+
+For this case, `spec.clusterSelector` will be ignored as `spec.clusterNames` is provided. This
+ensures that the results of dynamic scheduling have priority over manual definition of a
+cluster selector.
```

Comment:
This sentence is a bit confusing. The dynamic scheduling is the label based one, which *does not* have priority. And "manual" definition should refer to listing each cluster name, not selectors, right?

Reply:
'dynamic scheduling' in this instance is something like RSP or job scheduling - a mechanism that actively  to determines which clusters to propagate to and with what configuration.  Label selection is relatively static by comparison - it only considers the labels on a local cluster resources rather than the runtime state of the underlying cluster.

Would you like to suggest alternate terminology?

Reply:
It's clear to me now that I know the intention, and I think the terminology is fine for the intended meaning. The problem then is that I think it's quite easy to miss the change of context from the first sentence to the second.

Now that I think about it the reason for the label selector being ignored is that I assume RSP doesn't support labels yet. I think if that were mentioned here it would be much easier to read the second sentence as intended.

Reply:
Done by s/dynamically/runtime